packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -35,11 +35,17 @@ export interface Flags {
     dashboardAssetsGridMode: boolean; // dashboard UI
 }
 
+export enum BitcoinUnits {
+    BTC,
+    Sats,
+}
```

Comment:
you can use existing enum exported from connect:
```
import { PROTO } from '@trezor/connect';

PROTO.AmoutUnit

```

Reply:
done